tests/install_upgrade_operators/product_upgrade/utils.py

```
@@ -484,73 +488,67 @@ def get_all_cnv_alerts(prometheus, file_name, base_directory):
+
+    Uses an instant query with a PromQL range vector selector to retrieve all alert
+    samples since start_time, including alerts that have since resolved.
+
+    Args:
+        prometheus: Prometheus client instance.
+        start_time: Start of the lookback window.
+
+    Returns:
+        List of result dicts, each containing 'metric' labels and 'values' timestamps.
+    """
+    duration_seconds = int((datetime.now(tz=timezone.utc) - start_time).total_seconds())
```

Ruff (0.15.12)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)